robot/Cumulus/resources/GiftEntryPageObject.py

```diff
@@ -14,14 +14,17 @@
 class GiftEntryLandingPage(BaseNPSPPage, BasePage):
 
     
-    def _go_to_page(self):
-        """To go to Gift Entry page"""
+    def _go_to_page(self,tab=None):
+        """Go to Gift Entry page and wait until page contains batches table"""
         url_template = "{root}/lightning/n/{object}"
         name = self._object_name
         object_name = "{}{}".format(self.cumulusci.get_namespace_prefix(), name)
         url = url_template.format(root=self.cumulusci.org.lightning_base_url, object=object_name)
         self.selenium.go_to(url)
         self.salesforce.wait_until_loading_is_complete()
+        if tab is not None:
+            locator=npsp_lex_locators["gift_entry"]["id"].format("datatable Batches")                                           
+            self.selenium.wait_until_page_contains_element(locator)
```

Comment:
Can you please explain why this logic is needed ?

Reply:
made a slight change to code again. But the reason for this is gift entry page takes longer to load and if gift entry is enabled if displays batches table if not enabled shows error msg. So added a variable with default=error if so I can wait for that error msg in such cases and in other cases, it will wait for batches table to appear on the page.